bin/dump-to-git.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+var fs = require('fs');
+var process = require('process');
+var child_process = require('child_process');
+var Promise = require('bluebird');
+var NodeGit = require('nodegit');
+var LZString = require('lz-string');
+var models = require("./lib/models");
+
+var repoRootDir = './notes';
```

Comment:
Also, this directory has to be pre-existing and be a git repo, otherwise you run into 
`ENOENT: no such file or directory` or `Could not find repository from '.'`

I think the script should check for that and create the directory and init a git repo.